### PR TITLE
Add eslint setting 'flowDir' to specify location of .flowconfig

### PR DESCRIPTION
**Pull request by namjul**

### CODE REVIEW

**Comment by amilajack** [comment]
Seem to be a few failing test cases

**Comment by amilajack** [comment]
Would be great if you could add a few test cases. We use jest snapshot testing to make this easier.

**Comment by namjul** [comment]
Will do.

File: test/format.spec.js [hunk 1]
```diff
@@ -92,6 +93,11 @@ module.exports = {
     es6: true
   },
   plugins: ['flowtype-errors'],
+  "settings": {
+      "flowtype-errors": {
```
Comment (amilajack) [inline test/format.spec.js]: The indentation here seems to be off

**Comment by amilajack** [comment]
Also I think we should add some documentation to the readme on how to use this config setting

**Comment by namjul** [comment]
Yes, we probably should. I will add something.

**Comment by amilajack** [comment]
Just pushed `3.2.0-0`. Would be great if you could test this out!

```
npm install --save-dev eslint-pluigin-flowtype-errors@3.2.0-0
```

**Comment by namjul** [comment]
Nice, it works great for us.

**Comment by amilajack** [comment]
Just pushed `3.2.0` 🎉

**Comment by namjul** [comment]
So great thx!

**Pull request merged**